app/src/main/java/org/schulcloud/mobile/ui/feedback/FeedbackFragment.java

```diff
@@ -42,19 +44,25 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, view);
         Bundle args = getArguments();
 
-        mContextName = args.getString("contextName");
-        mCurrentUser = args.getString("currentUser");
+        mContextName = args.getString(ARGUMENT_CONTEXT_NAME);
+        mCurrentUser = args.getString(ARGUMENT_CURRENT_USER);
 
         mFeedbackPresenter.attachView(this);
-        mSendFeedback.setOnClickListener(v -> mFeedbackPresenter.sendFeedback(mEmail.getText().toString().trim(), mOpinion.getText().toString().trim(), mContextName, mCurrentUser));
+        mSendFeedback.setOnClickListener(v -> mFeedbackPresenter.sendFeedback(
+                getString(R.string.feedback_format),
+                mEmail.getText().toString().trim(),
+                mOpinion.getText().toString().trim(),
+                mContextName, mCurrentUser,
+                getString(R.string.feedback_subject),
+                getString(R.string.feedback_to)));
 
         return view;
     }
 
     @Override
     public void showContentHint() {
-        mOpinion.setHint("Bitte ausfüllen");
-        mOpinion.setError("Bitte ausfüllen");
+        mOpinion.setHint(R.string.feedback_hint_fillIn);
```

Comment:
Fixed. The problem is that setHint accepts resources and CharSequences, whereas setError only accepts CharSequences.